docs/excel/excel-add-ins-worksheets.md

```diff
@@ -313,6 +313,53 @@ function onWorksheetChanged(eventArgs) {
 }
 ```
 
+## Detect formula changes (preview)
+
+> [!NOTE]
+> The `Worksheet.onFormulaChanged` event is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+> 
+
+Your add-in can track changes to formulas in a worksheet. This may be useful when a worksheet is connected to an external database and formula changes in the worksheet need to trigger corresponding updates in the external database.
```

Comment:
A couple suggestions to make this sound more scenario-driven.
```suggestion
Your add-in can track changes to formulas in a worksheet. This is useful when a worksheet is connected to an external database. When the formula changes in the worksheet, the event in this scenario triggers corresponding updates in the external database.
```

Reply:
Thank you!